### ISSUE TITLE
Incorporate ReproNim p-hacking practical

### ISSUE THREAD

**Issue by emdupre** [reprocourse/reproducibility-neur-602#2]
For the [SfN 2018 ReproNim](https://www.eventbrite.ca/e/hands-on-reproducible-analysis-of-neuroimaging-data-nov-2-3-ucsd-tickets-49522133033#) course, @jbpoline created [a _p_-hacking practical](https://github.com/ReproNim/sfn2018-training/tree/master/section4/section40). It would be nice to integrate this into the ReproCourse materials !

**Comment by jbpoline** [comment]
good idea - Just a link to the repronim material or copying things over ?

**Comment by emdupre** [comment]
It totally is up to you ! My first reaction would be to reformat it as a non-notebook based practical, but I could see either way.
